.github/workflows/just-nuts.yml

```diff
@@ -22,13 +22,13 @@ jobs:
           - salesforcecli/plugin-community
           - salesforcecli/plugin-dev
           - salesforcecli/plugin-env
-          - salesforcecli/plugin-generate
           - salesforcecli/plugin-limits
           - salesforcecli/plugin-signups
           - salesforcecli/plugin-user
           - salesforcecli/plugin-packaging
           - salesforcecli/plugin-org
           - salesforcecli/plugin-auth
+          - salesforcecli/plugin-templates
           # omitted while nuts are too flaky on bulk commands
           # - salesforcecli/plugin-data
```

Comment:
these behave pretty well now from what I've seen.

Reply:
Please add schema, sobject and drop plugin login (since it's just a placeholder)  

then alphabetize to it's easier to compare the list to the oclif plugins prop on pjson